Tools/WebKitTestRunner/TestController.cpp

```diff
@@ -794,6 +794,11 @@ bool TestController::denyNotificationPermissionOnPrompt(WKStringRef originString
     return true;
 }
 
+void TestController::resetNotificationPermissions()
```

Comment:
Should probably be named removeAllWebNotificationPermissions.

Reply:
Fixed!